spec/requests/api/requests_spec.rb

```diff
@@ -246,14 +246,17 @@
       request.add_tag(t.name, t.children.first.name)
 
       api_basic_authorize action_identifier(:requests, :read, :resource_actions, :get)
-      run_get requests_url(request.id), :attributes => "workflow,v_allowed_tags,v_workflow_class"
+      run_get requests_url(request.id), :attributes => "workflow,v_allowed_tags,v_workflow_class,v_supports_pxe?,v_supports_iso?"
 
       expected_response = a_hash_including(
         "id"               => request.id,
         "workflow"         => a_hash_including("values"),
         "v_allowed_tags"   => [a_hash_including("children")],
         "v_workflow_class" => a_hash_including(
-          "instance_logger" => a_hash_including("klass" => request.workflow.class.to_s))
+          "instance_logger" => a_hash_including("klass" => request.workflow.class.to_s)
+        ),
+        "v_supports_pxe?"  => request.workflow.supports_pxe?,
```

Comment:
I'm wondering if this might better as a literal `true` or `false`, and, if it's appropriate, specify what they should be when building the request. That way, the test is less coupled to the implementation, and will also reveal any regressions (because this is saying we expect it to be whatever it happens to be, which will be true no matter what value it returns).